test/extended/templates/helpers.go

```diff
@@ -94,13 +94,14 @@ func EnsureTSB(tsbOC *exutil.CLI) (osbclient.Client, *exec.Cmd) {
 	if !exists {
 		e2e.Logf("Installing TSB onto the cluster for testing")
 		_, _, err := tsbOC.AsAdmin().WithoutNamespace().Run("create", "namespace").Args(tsbNS).Outputs()
-		o.Expect(err).NotTo(o.HaveOccurred())
+		// If template tests run in parallel this could be created twice, we don't really care.
+		fmt.Fprintf(g.GinkgoWriter, "Error creating TSB namespace %s: %v \n", tsbNS, err)
 		configPath := exutil.FixturePath("..", "..", "examples", "templateservicebroker", "templateservicebroker-template.yaml")
 		stdout, _, err := tsbOC.WithoutNamespace().Run("process").Args("-f", configPath).Outputs()
-		o.Expect(err).NotTo(o.HaveOccurred())
+		fmt.Fprintf(g.GinkgoWriter, "Error processing TSB template at %s: %v \n", configPath, err)
 		err = tsbOC.WithoutNamespace().AsAdmin().Run("create").Args("-f", "-").InputString(stdout).Execute()
-		// this is weird, but we have to ignore this error in case some of this already exists (race between tests and the like)
-		//o.Expect(err).NotTo(o.HaveOccurred())
+		// If template tests run in parallel this could be created twice, we don't really care.
+		fmt.Fprintf(g.GinkgoWriter, "Error creating TSB resources: %v \n", err)
```

Comment:
pretty sure it's not, but we use all sorts of varieties throughout the extended tests, I copied this from another one.  standardization isn't gonna happen here :)